binding.gyp

```diff
@@ -1,13 +1,13 @@
 # This file inherits default targets for Node addons, see https://github.com/nodejs/node-gyp/blob/master/addon.gypi
 {
   # https://github.com/springmeyer/gyp/blob/master/test/make_global_settings/wrapper/wrapper.gyp
-  'make_global_settings': [
-    ['CXX', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
-    ['CC', '<(module_root_dir)/mason_packages/.link/bin/clang'],
-    ['LINK', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
-    ['AR', '<(module_root_dir)/mason_packages/.link/bin/llvm-ar'],
-    ['NM', '<(module_root_dir)/mason_packages/.link/bin/llvm-nm']
-  ],
+  # 'make_global_settings': [
+  #   ['CXX', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
+  #   ['CC', '<(module_root_dir)/mason_packages/.link/bin/clang'],
+  #   ['LINK', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
+  #   ['AR', '<(module_root_dir)/mason_packages/.link/bin/llvm-ar'],
+  #   ['NM', '<(module_root_dir)/mason_packages/.link/bin/llvm-nm']
+  # ],
```

Comment:
If these are no longer needed, should they be removed?

Reply:
Yes, these are still needed. I had just mistakenly commented those out...